core/src/main/java/org/infinispan/functional/FunctionalMap.java

```diff
@@ -86,6 +86,15 @@
    ComponentStatus getStatus();
 
    /**
+    * Tells if the underlying cache is using encoding or not
+    *
+    * @return true if the underlying cache is encoded
+    */
+   default boolean isEncoded() {
```

Comment:
Looks like this isn't implemented/referenced anywhere. Seems we can remove. And even if it was, I believe _all_ caches are encoded now by default.

Reply:
yes, true. exactly. At first this was not the case but with @gustavonalle change this was not needed anymore. 
There is ticket to make encoding work properly with transactional caches too (it's not really implemented here) so I think I'm going to report your comments there and ensure they are done there if you agree